packages/ui/src/providers/DocumentInfo/index.tsx

```diff
@@ -260,6 +266,14 @@ const DocumentInfo: React.FC<
     [],
   )
 
+  useEffect(() => {
+    setInitialState(initialState)
+  }, [initialState])
+
+  useEffect(() => {
+    setInitialState(initialState)
```

Comment:
You have the identical `useEffect` functions written twice here.

Reply:
that is great